src/test/java/org/cactoos/collection/CollectionOfTest.java

```diff
@@ -34,11 +34,14 @@
 
 /**
  * Test Case for {@link CollectionOf}.
- *
+ * NOTE: This test class is to be removed
```

Comment:
@fanifieiev in my opinion this NOTE is duplicated by info in `@deprecated` section

Reply:
@iakunin OK